packages/ckeditor5-table/tests/tablecolumnresize/utils.js

```diff
@@ -574,6 +575,43 @@ describe( 'TableColumnResize utils', () => {
 			expect( result ).to.not.equal( 0 );
 		} );
 	} );
+
+	describe( 'setResizersEnablement()', () => {
+		let editor;
+
+		beforeEach( () => {
+			return ClassicEditor
+				.create( '', {
+					plugins: [ Table, TableColumnResize, Paragraph ]
+				} )
+				.then( newEditor => {
+					editor = newEditor;
+				} );
+		} );
+
+		afterEach( () => {
+			editor.destroy();
+		} );
+
+		it( 'should add and remove ck-resizers-hidden class to/from root', () => {
+			setModelData( editor.model, modelTable( [
+				[ '00', '01', '02' ],
+				[ '10', '11', '12' ]
+			], { columnWidths: '25%,25%,50%' } ) );
+
+			const view = editor.editing.view;
+
+			const viewRoot = view.domConverter.mapDomToView( view.getDomRoot() );
+
+			setResizersEnablement( view, false );
+
+			expect( viewRoot.hasClass( 'ck-resizers-hidden' ) ).to.be.true;
+
+			setResizersEnablement( view, true );
+
+			expect( viewRoot.hasClass( 'ck-resizers-hidden' ) ).to.be.false;
+		} );
```

Comment:
Currently this test checks two things. Ideally each unit test should check only one for easier problem detecting, so let's split it into two parts - first one adding and second one removing the class.

The second test should not repeat the steps from the first one (so setting the class on view by running the tested helper), but get the expected conditions right away. There is a `setData()` helper in `@ckeditor5-engine/src/dev-utils/view`, so let's use it here.

Reply:
Also it would be nice to cover the cases where nothing happens - so trying to remove the class if it's not applied and applying a class on the root that already has this class.